hls4ml/utils/attribute_descriptions.py

```diff
@@ -0,0 +1,46 @@
+"""Strings holding attribute descriptions."""
+
+# Common attributes
+
+reuse_factor = (
```

Comment:
Maybe worth saying something along the lines RF = 1 means all multiplications happen in parallel, i.e. one cc. Analog, a RF = k, corresponds to multiplications being spread out over k cc. Just to make it a bit more clear what the RF value can be set to.